client/src/component/Button/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button } from '@material-ui/core';
+
+const IconLabeledButton = ({ variant, color, icon, text, event, disable }) => (
+  <Button
+    variant={variant}
+    color={color}
```

Comment:
The color of the button will be primary or secondary !! , you need to add color to style to be flexible and change it to any color